Resources/config/parser.xml

```diff
@@ -19,5 +19,8 @@
         <service id="markdown.parser.flavored" class="Knp\Bundle\MarkdownBundle\Parser\Preset\Flavored" public="false">
             <tag name="markdown.parser" alias="flavored" />
         </service>
+
+        <!-- parsers will be injected with a compiler pass -->
+        <service id="markdown.parser.parser_manager" class="Knp\Bundle\MarkdownBundle\Parser\ParserManager" />
```

Comment:
this service should be marked as private, so that it can be inlined in case you don't use the PHP templating helper and the service definition of the helper gets removed (which won't happen for now as the helper service is always registered, even when the PHP templating is not there).
This service is not expected to be accessed dynamically